config/config_test.go

```diff
@@ -44,6 +46,20 @@ var _ = Describe("Config", func() {
 		})
 	})
 
+	Context("when invalid environment", func() {
+		It("returns an error", func() {
+			configJson := []byte(`{"account_name": "foo-account-name",
+								"account_key": "bar-account-key",
+								"container_name": "baz-container-name",
+								"environment": "invalid-cloud"}`)
+			configReader := bytes.NewReader(configJson)
+
+			config, err := config.NewFromReader(configReader)
+
+			Expect(err.Error()).To(Equal("unknown cloud environment: invalid-cloud"))
+			Expect(config.Environment).Should(BeEmpty())
+		})
+	})
 })
```

Comment:
I think it makes sense to add tests for the other environment as well, otherwise it looks good to me.

Reply:
makes sense!